e2e/testcases/resource_group_controller_test.go

```diff
@@ -53,53 +52,55 @@ func TestResourceGroupController(t *testing.T) {
 	nt.Must(rootSyncGitRepo.CommitAndPush("Adding a ConfigMap to repo"))
 	nt.Must(nt.WatchForAllSyncs())
 
-	// Checking that the ResourceGroup controller captures the status of the
-	// managed resources.
-	id := applier.InventoryID(configsync.RootSyncName, configsync.ControllerNamespace)
-	_, err := retry.Retry(60*time.Second, func() error {
-		rg := resourcegroup.Unstructured(configsync.RootSyncName, configsync.ControllerNamespace, id)
-		err := nt.Validate(configsync.RootSyncName, configsync.ControllerNamespace, rg,
-			testpredicates.AllResourcesReconciled(nt.Scheme))
-		if err != nil {
-			return err
-		}
-		return nil
-	})
-	if err != nil {
-		nt.T.Fatal(err)
-	}
+	nt.T.Log("Wait for the ResourceGroupController to update the ResourceGroup status")
+	nt.Must(nt.Watcher.WatchObject(kinds.ResourceGroup(), configsync.RootSyncName, configsync.ControllerNamespace,
+		testwatcher.WatchPredicates(
+			testpredicates.AllResourcesReconciled(nt.Scheme),
+		)))
 }
 
 func TestResourceGroupControllerInKptGroup(t *testing.T) {
 	nt := nomostest.New(t, nomostesting.ACMController)
 
+	// increase logging
```

Comment:
TODO: remove logging change after tests pass

Reply:
Meh. I'm just gonna leave in the verbose logging. It only gets written to the test log on test failure anyway, in which case we'll probably want verbose logging.